### PR TITLE
(RE) Update README.md

### PR DESCRIPTION
**Pull request by Kapestian**
This commit updates the README.md file to include most information about the project. The readme will continue to be updated as the project progresses.

This PR appears for the second time, so the process can be done in accordance with our group's guideline

### CODE REVIEW

**Review by SamSteu — COMMENTED** [review]
You can add examples of use cases for our application :
- meet new people when you are in a classroom
- meet with your neighbours

**Review by MaxCastelCast — APPROVED** [review]
We could maybe add also all the libreries and different tools that we are using for the project (like saying that we use firebase and code in Kotlin, etc..) for clarity purposes. But I think for now this is very clear and  it could be done later on.

**Review by BotondAKovacs — COMMENTED** [review]
Question: seems a bit weird to me that sometimes the third person is used: "people can identify common interests", "people in close proximity to discover each other" and sometimes the second person is used: "in your vicinity", "you have encountered".

**Comment by Kapestian** [comment]
> We could maybe add also all the libreries and different tools that we are using for the project (like saying that we use firebase and code in Kotlin, etc..) for clarity purposes. But I think for now this is very clear and it could be done later on.

@MaxCastelCast I thought about it but as we are not completely set with our choice of cloud services I was planning to add in the next sprint or even later.

**Comment by Kapestian** [comment]
> Question: seems a bit weird to me that sometimes the third person is used: "people can identify common interests", "people in close proximity to discover each other" and sometimes the second person is used: "in your vicinity", "you have encountered".

@BotondAKovacs You are right, i'll fix that

File: README.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
-# icebreakrr
+# Icebreakrr
+Welcome to our project: Icebreakrr, an application designed to facilitate first "In Real Life" contacts between people. We are a team of 7 students carrying out this project as part of an EPFL course. We are not accepting external collaborators at the moment.
```
[the remark this answers is not in the document]
Comment (Kapestian) [reply README.md:2]: The "In Real Life" is written that way so it stands out (as it is an important aspect of our app) and capitalizing the letters that way is a way to make the use of "IRL" more legit later, and thus the text more concise and readable. 
For the other comments, I made the appropriate changes. Thanks!

File: README.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+Icebreakrr uses the location of the phone to allow people in close proximity to discover each other and meet IRL.
+Each user creates a profile based on a photo, a catchphrase, a brief description and a few relevant tags, then shares it with all nearby users. By sharing profiles with nearby users, people can identify common interests, break the ice, and initiate conversations more naturally.
+
+Features (subject to change)
+
+- Account Creation & Multi-Device Sync: Create an account for seamless access across multiple devices.
+- Nearby User Discovery: Find profiles of people in your vicinity using real-time location data.
+- Profile Filtering: Sort and filter profiles based on tags, interests, or other parameters to find relevant matches.
+- Meet-up Requests: Ask for a meet-up via the app’s meeting request feature.
+- Offline Mode: Access and save profiles of people you have encountered, even when you’re offline.
```
[the remark this answers is not in the document]
Comment (Kapestian) [reply README.md:14]: You are right, I made it more clear

**Review by BotondAKovacs — APPROVED** [review]
LGTM

**Pull request merged**